src/libs/DistanceRequestUtils.ts

```diff
@@ -76,6 +78,27 @@ function getRoundedDistanceInUnits(distanceInMeters: number, unit: Unit): string
     return convertedDistance.toFixed(2);
 }
 
+/**
+ * @param hasRoute Whether the route exists for the distance request
+ * @param distanceInMeters Distance traveled
+ * @param unit Unit that should be used to display the distance
+ * @param rate Expensable amount allowed per unit
+ * @param translate Translate function
+ * @returns A string that describes the distance traveled
+ */
+function getDistanceForDisplay(hasRoute: boolean, distanceInMeters: number, unit: Unit, rate: number, translate: LocaleContextProps['translate']): string {
```

Comment:
Looks pretty good. @koko57, are we planning to use this function now when creating the merchant? I think so. If we don't use it now, when are we planning to start using it?

Reply:
Yes, we can use it to get the merchant - we can refactor the existing function for getting the merchant to use this one and the one I've created while working on the new rate field - I will take care of this refactor in my PR

Reply:
Ok thanks. @tienifr I think we should keep this util, and also use it within the function that gets the merchant in this PR. Then @koko57 can add the piece for the rate in her PR.